<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionssauvepoints</class>
 <widget class="QDialog" name="optionssauvepoints">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>211</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>File Path (default : keep empty)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Point Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Measures</string>
   </property>
  </widget>
  <widget class="QComboBox" name="NummeasBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>62</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>120</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3600</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cycle lenght (s)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CyclenBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Old points</string>
   </property>
  </widget>
  <widget class="QComboBox" name="OldpointBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>371</width>
     <height>40</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Remove Old Measures from the File</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Add the New Measures after the Old Content</string>
    </property>
   </item>
  </widget>
  <widget class="MyLineEdit" name="FilepathLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="FilepathButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="MyLineEdit" name="PointnameLine">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>270</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>SAVE POINT OPTIONS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>NummeasBox</zorder>
  <zorder>label_6</zorder>
  <zorder>CyclenBox</zorder>
  <zorder>label_7</zorder>
  <zorder>OldpointBox</zorder>
  <zorder>FilepathLine</zorder>
  <zorder>FilepathButton</zorder>
  <zorder>PointnameLine</zorder>
  <zorder>lineEdit</zorder>
  <zorder>SaveButton</zorder>
  <zorder>CancelButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
